src/features/email/component/email-sidebar/email-sidebar.tsx

```diff
@@ -39,22 +40,28 @@ interface EmailSidebarProps {
   handleCloseCompose: () => void;
   setSelectedEmail: (email: TEmail | null) => void;
   emails: Partial<TEmailData>;
```

Comment:
any reason to create two interface here I feel like we can separate the two different function as independent component file